ricoconverter/ricoconverter-convert/src/main/resources/xslt_eac/eac2rico-relations.xslt

```diff
@@ -46,6 +50,8 @@
 			<label>Relation de contrôle</label>
 			<shortcutIfSubjectIsSourceOfRelation>rico:isOrWasControllerOf</shortcutIfSubjectIsSourceOfRelation>
 			<shortcutIfSubjectIsTargetOfRelation>rico:hasOrHadController</shortcutIfSubjectIsTargetOfRelation>
+			<shortcutIfSubjectIsSourceOfRelation_current>rico:isOrWasControllerOf</shortcutIfSubjectIsSourceOfRelation_current>
```

Comment:
Nothing should change here. We do not have wasControllerOf or isControllerOf in RiC-O for now